source/usage-examples/deleteOne.txt

```diff
@@ -0,0 +1,97 @@
+.. _csharp-find-multiple:
+
+=================
+Delete a Document
+=================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can delete a document from a ``MongoCollection`` by using the synchronous
+`DeleteOne() <{+api-root+}/Overload_MongoDB_Driver_IMongoCollectionExtensions_DeleteOne.htm>`__
```

Comment:
We should add a note under the intro paragraph stating that DeleteOne() will delete the first document that it finds that matches the query filter. See the UpdateOne() usage example page for an example of this